os/arch/arm/src/armv7-m/up_usagefault.c

```diff
@@ -117,7 +117,7 @@ int up_usagefault(int irq, FAR void *context, FAR void *arg)
 	if (cfsr & INVPC) {
 		/* As PC value might be invalid use LR value to determine if
 		 * the crash occurred in the kernel space or in the user space */
-		system_exception_location = regs[REG_R14];
+		system_exception_location = regs[REG_R14] - 4;
```

Comment:
The above comment is wrong. INVPC does not mean that PC value is invalid. It means that there was an integrity check failure at the time of return from exception. The integrity check might fail due to various reasons. But, in any case, the PC value does not become invalid. So, I think we need to remove the whole if-statement.